settings.gradle.kts

```diff
@@ -125,3 +125,5 @@ include(":samples:05-file-transfer-cloud:consumer")
 include(":samples:05-file-transfer-cloud:provider")
 include(":samples:05-file-transfer-cloud:api")
 include(":samples:05-file-transfer-cloud:transfer-file")
+include("samples:06-prs:api")
+findProject(":samples:06-prs:api")?.name = "api"
```

Comment:
nit: this is probably not needed. Was it added by intellij? As I understand it just makes sure that the project is called "api".

Reply:
Yes it was added by intellij but yes it works without it